orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/testframework/utils/CucumberTestModule.java

```diff
@@ -55,6 +58,9 @@ public CucumberTestModule(CucumberTestState testState) {
     public void configure() {
         bind(CucumberTestState.class).toInstance(testState);
         bind(ProfileReader.class).to(CucumberProfileReader.class);
+        // The Cucumber tests do not use a schema version at all, but use the getProfile method of the
+        // CucumberTestModule to map the cucumber profile to its internal representation,
+        // so anything using the schema must use a no-op:
```

Comment:
i don't see why this comment is necessary.

or if wanted, i'd prefer a 1 line. ```// cucumber does not do schema validation```

Reply:
I added this detail because this tripped me up, meaning lots of cucumber tests failed without a helpful error message. It wasn't clear to me that I needed an implementation that did not do anything with the profile or schema files themselves, because those files are mocked out. Does this comment make that clear? If so, do you think this won't be useful for people doing this sort of thing?

Reply:
it might be clearer to bind them to mocked instances, rather than no op instances

not sure, 

but the comment is too wordy imo